avm/res/storage/storage-account/main.json

```diff
@@ -1673,8 +1673,8 @@
           "metadata": {
             "_generator": {
               "name": "bicep",
-              "version": "0.25.53.49325",
-              "templateHash": "774635646248081518"
+              "version": "0.25.3.34343",
```

Comment:
Please use latest bicep version to compile

Reply:
built it again - now with the newest version